bedrock/mozorg/templates/mozorg/home/includes/ijustine-promo.html

```diff
@@ -0,0 +1,37 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% call split(
+  image=resp_img(
+    url="img/home/2023/ijustine-promo/ijustine-promo.jpg",
+    srcset={
+      "img/home/2023/ijustine-promo/ijustine-promo-500.jpg": "500w",
+      "img/home/2023/ijustine-promo/ijustine-promo-700.jpg": "700w",
+      "img/home/2023/ijustine-promo/ijustine-promo-1000.jpg": "1000w",
+      "img/home/2023/ijustine-promo/ijustine-promo-1500.jpg": "1500w",
+    },
+    optional_attributes={
+          "class": "mzp-c-split-media-asset",
```

Comment:
**suggestion (a11y)**: add alt text?

Reply:
We don't usually add alt text to decorative images, though?

Reply:
@reemhamz Yeah, it's up to us to define what's decorative and I keep coming back to this piece from Léonie Watson: https://tink.uk/text-descriptions-emotion-rich-images/ 

Given this isn't a stock photo and there's not much copy associated with it, I would lean towards adding alt text in this case. The meaning I get from this photo is an emphasis not so much on the person, but the person being recorded. 

My rough draft: "iJustine, wearing jeans and a blue sweater with leopard print, speaks candidly with an interviewer who faces away from the camera. A microphone can be seen at the top of the frame and a video camera recording the interview takes up the left foreground." (all that said, this is your call in the end)

Reply:
Thanks for linking that blogpost! I hadn't thought of it like that since I started incorporating the _null alts with decorative images_ perspective. Appreciated the read, I see the benefit of adding the alt text here 😉 🔍